packages/proto-signing/package.json

```diff
@@ -44,6 +44,8 @@
     "postdefine-proto": "prettier --write \"src/generated/codecimpl.*\""
   },
   "dependencies": {
+    "@cosmjs/launchpad": "^0.22.0",
+    "@cosmjs/tendermint-rpc": "^0.22.0",
```

Comment:
- Do we really need launchpad?
- Shouldn't those be dev dependencies?

Reply:
We use launchpad for the wallet. It seems pretty convenient.